hackathon_site/dashboard/frontend/src/__tests__/Footer.test.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
It actually is a common convention from what I found. I personally am used to seeing tests separately, in non-UI code as well. 

I'll change it though - I can't think of a better reason than convention and finding tests easily is a good argument. I also don't really care and am sure the others aren't big on it as well.

Reply:
Also would prefer to have a separate `__tests__` folder but react-scripts actually doesn't look for tests outside `src/`, which is strange but wtv, maybe there's a setting to override it  but couldn't find it and too lazy

Reply:
Would definitely agree that if we were packaging and distributing just the react source as a package, it would make sense to have tests outside (Material UI does that, for example). But if they have to be in src, might as well make them easy to get to.

We probably can't do anything about `react-scripts` not looking outside of src without ejecting the app.